test/multigrid/amgx_pgm_kernels.cpp

```diff
@@ -164,9 +168,9 @@ class AmgxPgm : public ::testing::Test {
     }
 
     std::shared_ptr<gko::ReferenceExecutor> ref;
-    std::shared_ptr<const gko::CudaExecutor> cuda;
+    std::shared_ptr<gko::EXEC_TYPE> exec;
 
-    std::default_random_engine rand_engine;
+    std::ranlux48 rand_engine;
```

Comment:
I guess this should be reverted?
```suggestion
    std::default_random_engine rand_engine;
```

Reply:
good catch!